handlers/api.go

```diff
@@ -1357,6 +1359,7 @@ func validators(queryIndices []uint64) ([]interface{}, error) {
 		WHERE validatorindex = ANY($1)
 		GROUP BY validatorindex
 	) as w ON w.index = validators.validatorindex
+	LEFT JOIN (SELECT MAX(validatorindex)+1 FROM validator_performance WHERE validatorindex >= 0 AND validatorindex < 2147483647) validator_performance_count(total_count) ON true
```

Comment:
Doing this with `LEFT JOIN ... ON TRUE` is kinda weird, you can prob do this with a normal subquery.

Reply:
Good idea, this will increase readability without costing performance. Will do!